jujugui/static/gui/src/app/store/env/controller-api.js

```diff
@@ -1267,17 +1267,17 @@ YUI.add('juju-controller-api', function(Y) {
       Modify (grant or revoke) user access to the specified model.
 
       @method _modifyModelAccess
+      @param {String} modelId The UUID for the model on which the access is
+                              being changed.
       @param {Array} users The usernames of the users who need modified access.
-      @param {String} modelId The ID for the model on which the access is being
-                              changed.
+      @param {String} action Either 'revoke' or 'grant'.
       @param {String} access The level of access to grant the users; can be
                              'read', 'write', or 'admin'.
-      @param {String} action Either 'revoke' or 'grant'.
       @param {Function} callback A callable that must be called once the
         operation is performed. It will receive an error message or null if the
         access modification succeeded.
     */
-    _modifyModelAccess: function(users, modelId, access, action, callback) {
+    _modifyModelAccess: function(modelId, users, action, access, callback) {
```

Comment:
I had to change the signature around a bit in order to bind the modelId to the function before passing it into the Sharing component. While I was doing that, I decided I liked this signature a little better; it just "felt" better when coding.